### PR TITLE
Use Finviz to get similar companies based on Industry, Sector, Country

### PR DESCRIPTION
**Pull request by DidierRLopes**
* get command in Comparison Analysis menu to use Finviz as default. Much better than polygon. It filters by same Industry, Sector, and Country.
* new commands for brief overview,valuation,performance,ownership,technical,financial

E.g.
performance

### CODE REVIEW

File: gamestonk_terminal/comparison_analysis/ca_controller.py [hunk 1]
```diff
@@ -9,10 +9,12 @@
 from matplotlib import pyplot as plt
 from gamestonk_terminal import feature_flags as gtff
 from gamestonk_terminal import config_terminal as cfg
+from finvizfinance.screener.overview import Overview
 from gamestonk_terminal.helper_funcs import get_flair, parse_known_args_and_warn
 from gamestonk_terminal.comparison_analysis import yahoo_finance_api as yf_api
 from gamestonk_terminal.comparison_analysis import market_watch_api as mw_api
 from gamestonk_terminal.comparison_analysis import finbrain_api as f_api
+from gamestonk_terminal.comparison_analysis import finviz_view
```
Comment (jmaslek) [inline gamestonk_terminal/comparison_analysis/ca_controller.py:17]: If I get rid of this line (import finviz_view) it works

**Review by jmaslek — CHANGES_REQUESTED** [review]
Not sure if I am missing something but when I run terminal.py, I get `ImportError: cannot import name 'finviz_view' from "gamestonk_terminal.comparison_analysis" (unknown location)`

File: gamestonk_terminal/comparison_analysis/ca_controller.py (hunk 1 above)
Comment (DidierRLopes) [reply gamestonk_terminal/comparison_analysis/ca_controller.py:17]: This is the most important file of the PR, and I thought I had added it before the commit, but apparently I had not... It should work now (after I make sure there are no errors).

Comment (DidierRLopes) [reply gamestonk_terminal/comparison_analysis/ca_controller.py:17]: Thanks for checking this!

**Comment by jmaslek** [comment]
Haha that works much better :)

**Review by jmaslek — APPROVED** [review]

File: gamestonk_terminal/config_terminal.py [hunk 2]
```diff
@@ -19,7 +19,7 @@
 API_REDDIT_PASSWORD = os.getenv("GT_API_REDDIT_PASSWORD") or "REPLACE_ME"
 
 # https://polygon.io
-API_POLYGON_KEY = os.getenv("GT_API_POLYGON_KEY") or "REPLACE_ME"
+API_POLYGON_KEY = os.getenv("GT_API_POLYGON_KEY") or "pTx4m2fILimhK7YpR49Oa0DAec2bpnSG"
```
Comment (aia) [inline gamestonk_terminal/config_terminal.py:22]: You pushed your Polygon key :-)

**Pull request merged**